shpkpr/cli/options.py

```diff
@@ -110,11 +113,21 @@
 )
 
 
-template_file = click.option(
+template_path = click.option(
+    '--template_dir',
+    'template_path',
+    envvar="{0}_TEMPLATE_DIR".format(CONTEXT_SETTINGS['auto_envvar_prefix']),
```

Comment:
I'm OK with the more explicit `{0}` so unless there are more serious objections i'm going to leave this as is.